starter-core/src/main/java/io/micronaut/starter/feature/build/MicronautBuildPlugin.java

```diff
@@ -129,11 +131,21 @@ Optional<String> resolveRuntime(GeneratorContext generatorContext) {
                 .findFirst();
     }
 
+    @Nullable
+    private Set<String> ignoredAutomaticDependencies(GeneratorContext generatorContext) {
+        if (generatorContext.hasDependency(MicronautDependencyUtils.GROUP_ID_MICRONAUT_DATA, MicronautDependencyUtils.ARTIFACT_ID_MICRONAUT_DATA_TX_HIBERNATE)
+                && generatorContext.countDependencies(MicronautDependencyUtils.GROUP_ID_MICRONAUT_DATA) == 1) {
+            return Set.of(MicronautDependencyUtils.GROUP_ID_MICRONAUT_DATA + ":" + ARTIFACT_ID_MICRONAUT_DATA_PROCESSOR_ARTIFACT);
```

Comment:
I *think* this is breaking the hibernate-jpa guide for gradle https://github.com/micronaut-projects/micronaut-guides/pull/1369